ionic/src/app/about/about.page.html

```diff
@@ -1,7 +1,58 @@
 <ion-header>
   <ion-toolbar>
-    <ion-title>About</ion-title>
+    <ion-row class="title" center>
+      <ion-icon class="menu" name="menu"></ion-icon>
+      <ion-title>
+        Devices
+      </ion-title>
+    </ion-row>
   </ion-toolbar>
 </ion-header>
-
-<ion-content></ion-content>
+<ion-content padding>
+  <ion-row>
+    <div class="devices">
+      <h3>Your Devices</h3>
+    </div>
+  </ion-row>
+  <ion-button block expand="full" class="scan-devices"(click)="scanDevice()">Scan Devices</ion-button>
+  <div class="content">
+    <ion-card *ngFor="let chipId of user">
```

Comment:
*ngFor="let chipId of user" `chipId` from user ??
can you use appropriate variable names